packages/webdriver-utils/src/metadata/commonDesktopMetaDataResolver.js

```diff
@@ -0,0 +1,43 @@
+export default class CommonDesktopMetaDataResolver {
+  constructor(driver, opts = {}) {
+    this.driver = driver;
+    this.capabilities = opts;
```

Comment:
[Optional] change name, capabilities seems like it stores content of `driver.capabilities` which may or may not be true [ especially if you are planning for customer overrides ] 

And if its `capabilities` object then change from `opts` to `capabilities`

Reply:
- Add support for passing custom options from user like device name and version.